### DISCUSSION TITLE
Segmentation Fault in Llama.cpp server

### DISCUSSION THREAD

**Discussion by amyawu** [ggml-org/llama.cpp#8125] [Q&A]
I am not sure if there is a template to this and if so where to look.

I have a segmentation fault in trying to get the server to load. This is the error.

Can you help why it is doing this? I already killed all processes running in localhost; ty

INFO [                    main] build info | tid="140537152212992" timestamp=1719364332 build=3230 commit="163d50ad"
INFO [                    main] system info | tid="140537152212992" timestamp=1719364332 n_threads=1 n_threads_batch=-1 total_threads=2 system_info="AVX = 1 | AVX_VNNI = 0 | AVX2 = 1 | AVX512 = 1 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | AVX512_BF16 = 0 | FMA = 1 | NEON = 0 | SVE = 0 | ARM_FMA = 0 | F16C = 1 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 0 | SSE3 = 1 | SSSE3 = 1 | VSX = 0 | MATMUL_INT8 = 0 | LLAMAFILE = 1 | "

couldn't bind to server socket: hostname=127.0.0.1 port=8080

Segmentation fault

**Comment by micsthepick** [comment]
The segmentation fault is likely not intended, but can you check the output of `sudo netstat -plnt` if/when that happens again?
